frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDetails.tsx

```diff
@@ -84,7 +84,7 @@ const PipelineRunDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath,
     );
   }
 
-  if (!loaded || (!executionsLoaded && !executionsError)) {
+  if ((!loaded && !versionError) || (!executionsLoaded && !executionsError)) {
```

Comment:
use versionLoaded instead of versionError

Reply:
I think we don't need this check here. so removing it from here.